internal/pkg/cli/task_run_test.go

```diff
@@ -563,16 +563,22 @@ func TestTaskRunOpts_Execute(t *testing.T) {
 
 			opts := &runTaskOpts{
 				runTaskVars: runTaskVars{
+					GlobalOpts: &GlobalOpts{},
```

Comment:
nit: can we remove this?

Reply:
no because we need it to call `AppName()` in `deploy()` 😢